### DISCUSSION TITLE
Monitor container with name wildcard

### DISCUSSION THREAD

**Discussion by ITaluone** [containrrr/watchtower#1411] [Q&A] [answered]
Hi

Have a quick question: How do I monitor containers e.g. starting with "MyContainers.XYZ"?

I have found nothing in the documentation about that.

My use case is: I have a project (web api) in prod and dev, but the container name should have the version in it.. so I cannot simply list all containers for future :)

Ideally I would pass in `ServerXYZ.Dev*`, but how?

**Comment by piksel** [comment, marked as answer]
Support for specifying container names as regex patterns has been merged, but not yet released. You can try it in `containrrr/watchtower:latest-dev` in the meantime, but there isn't any documentation.
For your use case it would be `ServerXYZ\.Dev.*`

**Reply by ITaluone** [reply]
Thank you 👍
